src/widgets.js

```diff
@@ -3310,7 +3322,19 @@ PianoMenuMorph.prototype.init = function (
         'G# (68)' : 68,
         'B (71)' : 71,
         'Bb (70)' : 70,
-        'C (72)' : 72
+        'C (72)' : 72,
+        'D (74)' : 74,
+        'C# (73)' : 73,
+        'E (76)' : 76,
+        'Eb (75)' : 75,
+        'F (77)' : 77,
+        'G (79)' : 79,
+        'F# (78)' : 78,
+        'A (81)' : 81,
+        'G# (80)' : 80,
+        'B (83)' : 83,
+        'Bb (82)' : 82,
```

Comment:
Why Bb rather than A#? The rest of them (except `D#`) use `#` rather than `b`

Reply:
I did not change any of the conventions. It was already that way. I believe it is because Bb is the more common name for that note rather than A#.